server/chat-plugins/chatlog.ts

```diff
@@ -600,7 +600,7 @@ export abstract class Searcher {
 	) {
 		let buf = Utils.html`<div class="pad"><h2>Linecounts on `;
 		buf += `${roomid}${user ? ` for the user ${user}` : ` (top ${MAX_TOPUSERS})`}</h2>`;
-		buf += `<strong>Total lines: {total}</strong><br />`;
+		if (!user) buf += `<strong>Total lines: {total}</strong><br />`;
```

Comment:
``/linecount id=`` seems to be broken, shows {total} rather than the number. They are displayed below so my suggestion would be to hide this one when id= is provided, and leave it as is when there's no id

Current:
![image](https://user-images.githubusercontent.com/47090312/232739070-dd7dfb8c-2755-4b2e-8fa5-080c118fe00d.png)

Change:
![image](https://user-images.githubusercontent.com/47090312/232739599-925e296b-cba1-4e9d-922c-99bf12d9c48f.png)